docs/tutorial.md

```diff
@@ -1821,11 +1821,6 @@ import { user } from './user' // [!code ++]
 
 const app = new Elysia()
     .use(swagger())
-    .onError(({ error, code }) => {
```

Comment:
why did you remove this?

Reply:
Because this new code is mentioned in the following article

Reply:
There is no explanation for this part. The following is an explanation of this addition.